CommunicationServer/Application/Entities/GameRoom.cs

```diff
@@ -136,11 +158,11 @@ MoveTypes.Castling or
         };
     }
 
-    public IObservable<MoveMadeDto> GetMovesAsObservable()
+    public IObservable<ResponseJoinedGameDto> GetMovesAsObservable()
     {
-        var newMoves = Observable.FromEvent<MoveMadeDto>(
-            x => MoveReceived += x,
-            x => MoveReceived -= x);
-        return _moves.ToObservable().Concat(newMoves);
+        var newMoves = Observable.FromEvent<ResponseJoinedGameDto>(
+            x => GameJoined += x,
```

Comment:
That naming is confusing af. Is game joined happening many times?

Reply:
Yup, fixed it by renaming to JoinedGameStream